src/packagedcode/rubygems.py

```diff
@@ -648,92 +628,163 @@ def normalize(gem_data, known_fields=known_fields):
+        if '.version ' in individual:
+            version = individual.split('= ')
+            version = version[1].strip('\'"%q{}')
+            gemspec_data['version'] = version
+
+        # update the value of author
+        if '.authors ' in individual:
+            # >>> s = 's.authors = ["abc", "pqr", "xyz"]'
+            # >>> result = s.split('= ')
+            # ['s.authors ', '["abc", "pqr", "xyz"]']
+            # >>> result[1].strip('[]')
+            # '"abc", "pqr", "xyz"'
+            authors = individual.split('= ')
+            authors = authors[1].strip('[]%q{}').replace('"', '')
+            gemspec_data['authors'] = authors
+
+        # update the value of email
+        if '.email ' in individual:
+            email = individual.split('= ')
+            email = email[1].strip('[]%q{}').replace('"', '')
+            gemspec_data['email'] = email
+
+        # update the value of summary
+        if '.summary ' in individual:
+            summary = individual.split('= ')
+            summary = summary[1].strip('\'"%q{}')
+            gemspec_data['summary'] = summary
+
+        # update the value of description
+        if '.description ' in individual:
+            description = individual.split('= ')
+            description = description[1].strip('\'"%q{}')
+            gemspec_data['description'] = description
+
+        # update the value of homepage
+        if '.homepage ' in individual and 'homepage_uri' not in individual:
+            homepage = individual.split('= ')
+            homepage = homepage[1].strip('\'"%q{}')
+            gemspec_data['homepage'] = homepage
+
+        # update the value of license
+        if '.license ' in individual:
+            license = individual.split('= ')
+            license = license[1].strip('\'"%q{}')
+            gemspec_data['license'] = license
+
+        # update the value of dependencies
+        if 'dependency' in  individual:
+            dep_version = individual
+            # >>> s = 's.add_runtime_dependency(%q<mechanize>, [">= 0"])'
+            # >>> z = re.split(", ", s)
+            # >>> z = z[1].strip("\"'[]()")
+            # >>> z
+            # '>= 0'
+            dep_version = re.split(", ", dep_version)
+            try:
+                dep_version = dep_version[1].strip(" \"'[]()")
+            except IndexError:
+                dep_version = None
+
+            # >>> s = 's.add_runtime_dependency(%q<mechanize>, [">= 0"])'
+            # >>> d = re.split("'|<|>", s)
+            # >>> d
+            # ['s.add_runtime_dependency(%q', 'mechanize', ', ["', '= 0"])']
+            dependency = re.split("'|\"|<|>", individual)
+            dependencies[dependency[1]] = dep_version
+
+    # stores dependencies list into dictionary
+    gemspec_data['dependencies'] = dependencies
+
+    return gemspec_data
+
+
+def build_packages_from_gemspec(gemspec_data):
     """
-    Represent a Gem specification.
+    Return a package built from gemspec.
```

Comment:
What is `gemspec` ? instead say:
`Return a RubyGem package built from a `gemspec_data` mapping.`